misc/python/materialize/cloudtest/app/materialize_application.py

```diff
@@ -208,3 +219,23 @@ def get_cluster_and_replica_id(self, mz_table: str, name: str) -> tuple[str, str
             f"SELECT s.cluster_id, r.id FROM {mz_table} s JOIN mz_cluster_replicas r ON r.cluster_id = s.cluster_id WHERE s.name = '{name}'"
         )[0]
         return cluster_id, replica_id
+
+    def suspend_k8s_node(self, node_name: str) -> None:
+        print(f"Suspending node {node_name}...")
+        result = subprocess.run(
```

Comment:
If a KinD node is paused and then a test fails, the next test is going to find itself in a degraded KinD environment. There would be two ways out of this situation:
- have pytest run `unpause` on all nodes as part of any testcase teardown
- just run each of the 4 tests in a separate CI step.

Reply:
Actually in the CI we already run pytest with the option to fail on the first failure, so it is OK to leave the cluster degraded.